--- a/dist/index.js
+++ b/dist/index.js
@@ -111,7 +111,7 @@
     const loader = typeof init === "function" ? init : init?.loader;
     const wasm = typeof init === "function" ? undefined : init?.wasm;
     const module = loader ? await loader() : await defaultLoader();
-    await module.default(wasm);
+    await module.default({ module_or_path: wasm });
     wasmModule = module;
     return module;
 }
--- a/src/index.ts
+++ b/src/index.ts
@@ -267,7 +267,7 @@
   const loader = typeof init === "function" ? init : init?.loader;
   const wasm = typeof init === "function" ? undefined : init?.wasm;
   const module = loader ? await loader() : await defaultLoader();
-  await module.default(wasm);
+  await module.default({ module_or_path: wasm });
   wasmModule = module;
   return module;
 }
